docs/reference/run.md

```diff
@@ -190,6 +190,8 @@ Images using the v2 or later image format have a content-addressable identifier
 called a digest. As long as the input used to generate the image is unchanged,
 the digest value is predictable and referenceable.
 
+    docker run alpine@sha256:9cacb71397b640eca97488cf08582ae4e4068513101088e9f96c9814bfda95e0 date
```

Comment:
Thanks! Can you add a short explanation of the example itself? Something like;

```markdown
The following example runs a container from the `alpine` image
with the `sha256:9cacb71397b640eca97488cf08582ae4e4068513101088e9f96c9814bfda95e0`
digest:

    $ docker run alpine@sha256:9cacb71397b640eca97488cf08582ae4e4068513101088e9f96c9814bfda95e0 date
```

Note that for shell examples, we alway include the prompt (`$ `)